src/features/defi/providers/osmosis/components/OsmosisManager/Withdraw/components/Status.tsx

```diff
@@ -54,6 +54,7 @@ export const Status: React.FC<StatusProps> = ({ accountId }) => {
   const userAddress = useMemo(() => accountId && fromAccountId(accountId).account, [accountId])
 
   const serializedTxIndex = useMemo(() => {
+    // TODO:pastaghost): refactor to eliminate userAddress parameter
```

Comment:
Nice one, we may or may not (probably not, see below) want to refactor `serializeTxIndex` to speak full CAIP at some point. For context, the rationale to it speaking `userAddress` in the last arg is because an xpub (an AccountId in our terminology) may have different addresses, and the Tx needs both the `AccountId` *and* the xpub, currently serialized using the address only, not a CAIP.

Not sure if that'll ever change, because `AccountId` carries a mental model accross the codebase of being a "portfolio account", or higher-level account. Not e.g a derived address from an xpub, nor e.g a CAIP representing an "upstream address" (a smart contract, another user's EOA, a validator address etc).